src/Randomizer.SMZ3.Tracking/Configuration/ConfigFiles/ItemConfig.cs

```diff
@@ -941,7 +959,7 @@ public static ItemConfig Default()
                     Item = "Missile",
                     Name = new("Missile", new("Missiles", 0), new("Missile pack", 0.2)),
                     InternalItemType = ItemType.Missile,
-                    Plural = new("Missiles", new("Missile packs", 0)),
+                    Plural = new("Missile", new("Missile pack", 0), new("Missile", 0)),
```

Comment:
Was this intentional?

Reply:
Ah, thanks for catching that. Was having issues with tracker saying "Missiles-is" basically, but that turned out to be an issue with one of the lines in ResponseConfig.cs.